### ISSUE TITLE
Gender Score - Obscuring meaningful differences?

### ISSUE THREAD

**Issue by AlexanderTyan** [uchicago-computation-workshop/ben_zhao#21]
It seems like Gender Target score would balance out to 0 in two meaningfully different cases:
1. If the language in a job listing is a "little" bit masculine in "little" bit feminine. => Neutral score.
AND
2. If the language in a job listing is extremely polarized. I.e. lots of masculine words, but also an equal amount of feminine words; still averaging out to "neutral."

Would the way the score is calculated obscure these differences, calculating both scenarios to be "neutral?" What if that matters to, say, a potential applicant who may react quite differently in these two different scenarios? (Though I suppose, the end conclusion of the paper is that applicants do not react to the text content all that much.)

**Comment by bensoltoff** [comment]
I wonder how likely situation 2 occurs in the data. I can see the first scenario occurring regularly, but it would make more sense for job applications to be extreme in one of the directions, not in both. What kind of application would use lots of extreme feminine and masculine words?

**Comment by AlexanderTyan** [comment]
Hmm, I suppose I should clarify that the question is getting at the idea that two listings with nearly identical Gender Target scores may have substantially different gender "polarizations." 1 and 2 are just two extremes. Scenario two can be though of as "extreme" compared to 1, not in an absolute sense. But in either case, wouldn't the calculation obscure such potentially meaningful differences?

**Comment by bensoltoff** [comment]
You're right, it would with a pure average. Incorporating a measure of deviation would help to detect how extreme the values would be. But I guess my comment is that I don't get the practical impact of this on the findings, unless there are a substantial number of job postings that have the extreme polarization you describe. Obviously looking at the data would quickly clear this up (which we don't have), but I would think extreme polarization like you describe is a rare occurrence.
